src/miner.h

```diff
@@ -180,7 +180,7 @@ class BlockAssembler
     /** Add transactions based on feerate including unconfirmed ancestors
       * Increments nPackagesSelected / nDescendantsUpdated with corresponding
       * statistics from the package selection (for logging statistics). */
-    void addPackageTxs(int &nPackagesSelected, int &nDescendantsUpdated);
+    void addPackageTxs(int &nPackagesSelected, int &nDescendantsUpdated, int required_age_in_secs=0);
```

Comment:
Do we need the default value here too? It defaults to 0 on the rpc already.
Same for CreateNewBlock although I guess there we do need it for generate.

Reply:
I'm just passing it through, since we only want to use it with "getnewblockhex" type stuff and not generate.

Reply:
But only with the default value in CreateNewBlock should be enough, no?